crd/multitenancy/manifests/multitenancy.acn.azure.com_podnetworks.yaml

```diff
@@ -80,6 +80,7 @@ spec:
                 - Ready
                 - InUse
                 - SubnetNotDelegated
+                - SubnetDelegationValidateFailed
```

Comment:
Will update/ fix this to match podnetwork.go. Currently having issues pushing code.